contrib/ci/Dockerfile-fedora.in

```diff
@@ -1,4 +1,4 @@
-FROM fedora:32
+FROM fedora:34
```

Comment:
so this won't be affected until "tomorrow" if it's merged since we only build containers once a day.

Reply:
I'm cack-handedly trying to debug the Fedora CI issue of `meson.build:1:0: ERROR: Compiler cc can not compile programs` -- I actually think the `FROM fedora:34` is being ignored as `main.yml` is specifying `fwupd-${{matrix.os}}:latest` -- I wondered if the odd timing of the failure to coincide with the F34 release, but looking at https://hub.docker.com/_/fedora the `latest` is still listed as f33.

Reply:
>I'm cack-handedly trying to debug the Fedora CI issue of meson.build:1:0: ERROR: Compiler cc can not compile programs -- I actually think the FROM fedora:34 is being ignored as main.yml is specifying fwupd-${{matrix.os}}:latest -- I wondered if the odd timing of the failure to coincide with the F34 release, but looking at https://hub.docker.com/_/fedora the latest is still listed as f33.

[build_diff.txt](https://github.com/fwupd/fwupd/files/6386266/build_diff.txt)

Here is the difference from the two container builds from day to day.   I don't see it pulling F34 in the diff.  But there are a bunch of packages that upgraded and presumably one of those caused the failure.

Reply:
And FYI I got those from https://github.com/fwupd/fwupd/actions/workflows/create_containers.yml

Reply:
Interesting, good detective work! I guess the main ones of interest would be glibc, and rpm, the latter even more likely as it only seems to be happening in rpmbuild. I guess there's no way to test these as doing so would mean merging and waiting 24h for the CI to regenerate?

I tested this fedora:34 version with podman locally, and it worked, so I'm pondering merging this patch and seeing if the problem magically goes away in 24h? Insane?

Reply:
I mean this seems like a real problem on fedora 32 no? Like anyone on Fedora 32 should be hitting it and able to reproduce it.

Reply:
> I tested this fedora:34 version with podman locally, and it worked, so I'm pondering merging this patch and seeing if the problem magically goes away in 24h? Insane?

I wonder if there is a way to trigger container build on demand? That's really what we want.

As for moving to Fedora 34, I don't see a problem moving, 32 will eventually hit EOL anyways.

Reply:
I think you can just re-run the build containers job here: https://github.com/fwupd/fwupd/runs/2443676441?check_suite_focus=true if someone from Fedora thinks it's been fixed in F32.

Reply:
I think F32 is kinda EOL now, so I don't really want to spent the time on tracking it down.